src/util/api.ts

```diff
@@ -0,0 +1,11 @@
+const getUserLocation = (accessToken: any) => {
+  return fetch(`https://ipinfo.io/json?token=${accessToken}`)
+    .then((response) => {
+      return response.json();
+    })
+    .then((jsonResponse) => {
```

Comment:
Why this additional `.then`?

Reply:
Actually no need for that. Removed and will recommit the change

Reply:
Cool, feel free to merge then!